Source/Plugins/CoherenceViewer/CoherenceNode.cpp

```diff
@@ -173,70 +173,30 @@ void CoherenceNode::updateDataBufferSize(int newSize)
     int totalChans = nGroup1Chans + nGroup2Chans;
 
     // no writers or readers can exist here
-    // so this probably can't be called during acquisition
-    // I changed it because it doesn't really make sense to just change the buffer we're currently
-    // writing to when acquisition is stopped, and we need to figure out how to
-    // make changes when acquisition is running anyway.
+    // so this can't be called during acquisition
```

Comment:
Realized this and the other resize function were way too complicated for what they were doing (mostly my fault!)